src/gobmessage/hr/kvk/message.py

```diff
@@ -66,17 +66,23 @@ def _process_entity(self, source: dict, mapper: Mapper) -> list[UpdateObject]:
 
         if isinstance(mapper, MaatschappelijkeActiviteitenMapper):
             update_objects += self._get_vestigingen(mapper.get_vestigingsnummers(mapped_entity))
+        elif isinstance(mapper, VestigingenMapper):
+            update_objects += self._get_locaties(mapper.get_locaties(source))
```

Comment:
Why is this signature different than the one above (mapped_entity vs source)? This is a bit confusing.